v4l2-request-test.c

```diff
@@ -257,7 +266,26 @@ int main(int argc, char *argv[])
 	setup_config(&config);
 
 	while (1) {
-		opt = getopt(argc, argv, "v:m:d:D:s:f:P:ilqh");
+	        int option_index = 0;
+		static struct option long_options[] = {
+						       {"device",        required_argument, 0, 'v' },
```

Comment:
Please add a whitespace between { and the first string in each declaration.

Reply:
done.